tic_tac_toe/game.py

```diff
@@ -63,42 +78,106 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    
+    set_of_combos = ( #row wins
+            ((0,0), (0,1), (0,2)),    
+            ((1,0), (1,1), (1,2)),    
+            ((2,0), (2,1), (2,2)),
+            #column wins
+            ((0,0), (1,0), (2,0)),
+            ((0,1), (1,1), (2,1)),
+            ((0,2), (1,2), (2,2)),
+            #diagonal wins
+            ((0,0), (1,1), (2,2)),
+            ((2,0), (1,1), (0,2))
+            )
+    
+    for combo in set_of_combos:
+        if _is_winning_combination(board, combo, player):
+            return player
+    return None
 
 # public interface
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
-
+    game = {
+        'player1': player1,
+        'player2': player2,
+        'board': [
+            ["-", "-", "-"],
+            ["-", "-", "-"],
+            ["-", "-", "-"],
+        ],
+        'next_turn': player1,
+        'winner': None
+    }
+    return game
 
 def get_winner(game):
     """
     Returns the winner player if any, or None otherwise.
     """
-    pass
-
+    return game['winner']
 
 def move(game, player, position):
     """
     Performs a player movement in the game. Must ensure all the pre requisites
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
-
+    board = game['board']
+    #check to see if the right player is the one making the move    
+    if player != game['next_turn']:
+        raise InvalidMovement("\""+ game['next_turn'] + "\"" +" moves next.")
```

Comment:
For next time, I'd suggest using string formatting here instead of building it up like this. This method works, but using the other way is much easier to read, imho